packages/blocks/src/list-block/list-block.ts

```diff
@@ -102,21 +115,28 @@ export class ListBlockComponent extends NonShadowLitElement {
       deep,
       index,
       onClick: () => {
-        if (this.model.type !== 'todo') {
-          selectList(this.model);
+        if (this.model.type === 'todo' || this.model.type === 'toggle') {
```

Comment:
should `toggle` be saved in the crdt?

Reply:
I think we all agree that the answer here is, no it should not. So I'll go ahead and resolve this.